core/arch/x86/opcode.h

```diff
@@ -1319,6 +1319,13 @@ enum {
     /* 1156 */ OP_ktestq,   /**< IA-32/AMD64 AVX-512 ktestd opcode. */
     /* 1157 */ OP_ktestd,   /**< IA-32/AMD64 AVX-512 ktestd opcode. */
 
+    /* 1158 */ OP_vmovdqu8,  /**< IA-32/AMD64 AVX-512 vmovdqu8 opcode. */
```

Comment:
What is the plan for how to order these?  Maybe they should be alphabetical within the ISA group.  The ones above are sorted by encoding: is that what you're doing here?  Let's have a plan with a meaningful enum ordering rather than random.

Reply:
I've re-sorted them alphabetically, let's do that from now on.